### PR TITLE
Convert wkt types to geo types

### PR DESCRIPTION
**Pull request by datanel**
just need review
do not merge please.

I need it for https://github.com/CanalTP/navitia_model/pull/101: (de)serialize wkt string <=> geo type

### CODE REVIEW

**Comment by datanel** [comment]
@TeXitoi @benoit-bst @xlqian @pbougue anyone can review this PR please?

File: src/conversion.rs [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+            .0
+            .iter()
+            .map(|p| w_polygon_to_g_polygon(p))
+            .collect();
+
+        geo_multipolygon.into()
+    }
+}
+
+pub fn try_into_geometrycollection(
```
Comment (nonifier) [inline src/conversion.rs]: Maybe `try_collection_into_geometry` would be better

File: src/conversion.rs [hunk 2]
```diff
@@ -0,0 +1,682 @@
+// Copyright 2014-2015 The GeoRust Developers
```
Comment (TeXitoi) [inline src/conversion.rs]: update date

File: src/conversion.rs [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+        .iter()
+        .map(|l| l.0.iter().map(|c| (c.x, c.y)).collect::<Vec<_>>().into())
+        .collect();
+
+    let interior: geo_types::LineString<f64>;
+    if !geo_linestrings.is_empty() {
+        interior = geo_linestrings.remove(0);
+    } else {
+        interior = geo_types::LineString(vec![]);
+    }
```
Comment (TeXitoi) [inline src/conversion.rs]:
```rust
let mut iter = polygon
    .0
    .iter()
    .map(|l| l.0.iter().map(|c| (c.x, c.y)).collect::<Vec<_>>().into());
match iter.next() {
    Some(interior) => geo_types::Polygon::new(interior, iter.collect()),
    None => geo_types::Polygon::new(vec![], vec![]),
}

File: src/conversion.rs (hunk 1 above)
Comment (TeXitoi) [inline src/conversion.rs]: `try_into_geometry_collection`

**Comment by datanel** [comment]
this pr sas merged into thé original repo

**Pull request closed without merging**